cmake/ctest/drivers/geminga/ctest_linux_nightly_mpi_release_tpetra_no_int_serial_experimental_geminga.cmake

```diff
@@ -54,28 +54,32 @@
 # @HEADER
 
 
-INCLUDE("${CTEST_SCRIPT_DIRECTORY}/TrilinosCTestDriverCore.geminga.gcc-broken.cmake")
+INCLUDE("${CTEST_SCRIPT_DIRECTORY}/TrilinosCTestDriverCore.geminga.gcc.cmake")
 
 #
 # Set the options specific to this build case
 #
 
+# The variable BUILD_DIR_NAME is based COMM_TYPE, BUILD_TYPE, and BUILD_NAME_DETAILS.
+# Tribits creates the variable listed under "Build Name" by prepending the OS type and compiler
+# details to BUILD_DIR_NAME.
 SET(COMM_TYPE MPI)
 SET(BUILD_TYPE RELEASE)
-SET(BUILD_DIR_NAME BROKEN_NO_SERIAL)
+SET(BUILD_NAME_DETAILS NO-EPETRA_NO-INT_SERIAL_EXPERIMENTAL)
```

Comment:
Why does the name have `SERIAL` in it?

Reply:
Because its a SERIAL (not OMP) build.  Or do we presume that it is the default and only label OPENMP & CUDA?